README.md

```diff
@@ -53,60 +99,25 @@ Word Processors which can generate LaTeX, HTML or PDF output on demand.
 - [nbconvert](https://nbconvert.readthedocs.io/en/latest/) - Convert Jupyter
    notebooks into `reveal.js` presentations, PDF, HTML, Markdown,
    reStructuredText and more.
-- [org-mode](https://orgmode.org) - Powerful Emacs package for authoring notes,
-   TODO lists, spreadsheets, documents, executable code-blocks and a lot more
-   in a plain text format called Org :bookmark: :link:.
 - [pandoc](https://pandoc.org/MANUAL) - A Haskell library for converting from
    one markup format to another, and a command-line tool that uses this
    library :bookmark: :link:.
-- [Zettlr](https://www.zettlr.com/) - An electron based markdown editor which
-   integrates Zotero, pandoc and many more features wrapped in an elegant WYSIWYG
-   interface :bookmark: :link:.
-
-## Bibliography
-Bibliography managers to generate citations / BibTeX / BibLaTeX files.
-
-- [Better BibTeX for Zotero](https://retorque.re/zotero-better-bibtex/) - Enhanced
- exporting tool for Zotero.
-- [Citation Style Language (CSL) styles](https://editor.citationstyles.org/) -  A
- crowdsourced repository with over 9000 free CSL citation styles and an online
- editor to create new ones.
-- [fzf-bibtex](https://github.com/msprev/fzf-bibtex/#readme) - A BibTeX source
- with vim integration which uses fzf (a fuzzy finder implemented in Go).
-- [Zotero](https://www.zotero.org/) - FOSS tool to collect, organize, cite, and
- share research.
-
-## Illustrations
-Drawing illustrations themselves has driven many a scientist mad. Fortunately,
-there are formal languages with which one can create beautiful graphics.
-
-- [diagrams.net](https://www.diagrams.net/) - Open source, online, desktop and
- container deployable diagramming software.
-- [graphviz](https://graphviz.org/) - Visualization software for graphs and
- networks which uses a domain-specific DOT language.
-
-## Extras
-Supplementary files and tools.
-
 - [Pandoc filters](https://github.com/jgm/pandoc/wiki/Pandoc-Filters) - List of
  addons to pandoc which implement extra features such as citations and
  cross-references.
 - [Panflute](http://scorreia.com/software/panflute/) - A pythonic alternative
  to John MacFarlane's pandocfilters.
-- [vim-pandoc](https://github.com/vim-pandoc/vim-pandoc/#readme) - Pandoc
- integration and utilities for Vim.
-- [vim-pandoc-syntax](https://github.com/vim-pandoc/vim-pandoc-syntax/#readme) - Lightweight
- rendering of pandoc syntax in Vim.
 
-## Spell Checking
-Check spelling.
-
-## Linters
-Check grammar and language.
+## Spell Checking and Linting
 
+- [GNU Aspell](http://aspell.net/)
+- [Hunspell](http://hunspell.github.io/)
 - [LanguageTool](https://languagetool.org/) - Open source grammar, style and
  spell Checker.
+- [Markdown lint tool](https://github.com/markdownlint/markdownlint)
 - [proselint](http://proselint.com/) - A linter for prose.
+- [remarklint](https://github.com/remarkjs/remark-lint)
+- [restructuredtext-lint](https://github.com/twolfson/restructuredtext-lint)
```

Comment:
Add a small description per tool as you see fit.

Reply:
I will add a description asap.